plugins/autowarmth.koplugin/main.lua

```diff
@@ -533,6 +533,7 @@ function AutoWarmth:getLocationMenu()
                 wrap = false,
                 value_step = 10,
                 value_hold_step = 100,
+                -- @translators This is the unit for meter.
                 unit = _("m"),
```

Comment:
Context to disambiguate is likely still required regardless while also being self-explanatory.